src/plugins/intel_gna/tests/deprecated/helpers/test_model_path.cpp

```diff
@@ -6,7 +6,7 @@
 // Created by user on 19.10.18.
 //
 
-#include "test_model_repo.hpp"
+#include "helpers/test_model_repo.hpp"
```

Comment:
I am not sure if I understand correctly
The files 
test_model_repo.hpp and test_model_repo.cpp are moved from deprecated tests to functional test and now deprecated tests are dependent on the code of functional tests?

Reply:
Yes, you got it right